engine/src/main/java/org/destinationsol/ui/Position.java

```diff
@@ -29,9 +29,9 @@ public Position(float xNormalized, float yNormalized) {
         set(xNormalized, yNormalized);
     }
 
-    public void set(float xNormalized, float yNormalized) {
```

Comment:
There is a setting in Checkstyle to allow for name shadowing in setters and constructors. Maybe this should be enabled for DestSol to avoid cases like this (I think it is completely fine to shadow names in theses cases, but I don't decide this ^^)

Reply:
Personally, I would preferred to have done the same, however the checkstyle rules appear to dictate that it is invalid, rather than just warning the user.

The setting is `HiddenField.ignoreSetter` and it is off by default. The configuration is shared between both Destination Sol and Terasology though, so any changes made would affect both projects.
##### (source: https://checkstyle.sourceforge.io/config_coding.html#HiddenField_Properties)
It does seem that this option is already enabled (see [here](https://github.com/MovingBlocks/TeraConfig/blob/44c5632fff7ac493fc4cbc082731dd92b94c01ad/checkstyle/checkstyle.xml#L229)), so should I revert the parameter name changes? Would it be acceptable to add a `SuppressWarnings` annotation on these methods instead?

I think that the reason why they are not caught by the setter exception rule is because they both have more than one parameter and are called `set`, rather than `set<name>`.

Reply:
> I think that the reason why they are not caught by the setter exception rule is because they both have more than one parameter and are called set, rather than set<name>.

That makes sense. In this case, suppressing the warning might be okay here with a remark that it is basically the same as parameter shadowing in setters (which we allow).

Reply:
I have reverted the variable name changes and added a `SuppressWarning` annotation. The warnings may be visible again as the checkstyle rules do not appear to detect such annotations with the current configuration. This may require a `SuppressWarningsFilter` and a `SupressWarningsHolder` to be added to the configuration file (which is stored in the [TeraConfig](https://github.com/MovingBlocks/TeraConfig) repository).
##### (source: https://checkstyle.sourceforge.io/config_filters.html#SuppressWarningsFilter)